src/Behat/Config/Config.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Behat\Config;
+
+final class Config
+{
+    public function __construct(
+        private array $settings = []
+    ) {
+    }
+
+    public function import(string $file): self
```

Comment:
Could be interesting...

```suggestion
    public function import(... string $files): self
```